src/scatter_buildblock/ScatterSimulation.cxx

```diff
@@ -230,7 +232,7 @@ ScatterSimulation::set_defaults()
     this->zoom_size_xy = -1;
     this->zoom_size_z = -1;
     this->downsample_scanner_bool = false;
-    this->downsample_scanner_dets = 64;
+    this->downsample_scanner_dets = -1;
```

Comment:
could this be the reason of the slow-down? we may need to force it to 64 in the parfile

Reply:
it does make the test way faster

Reply:
It seems to point to an error in the code where you set it to 64 if it is -1. I wouldn't want to have a default that is essentially unworkable.

Reply:
sorry not sure I follow

Reply:
resolved in offline discussions